src/app/app.routes.ts

```diff
@@ -95,6 +98,10 @@ export const routes: Routes = [
     component: SecretSelectionPageComponent,
   },
 
+        {
+    path: 'creationPersonnage/races',
```

Comment:
use lower case and hyphen rather than camel case for urls (saw it in other pr's maybe consider refactoring when dev is updated

Reply:
OK i changed the both paths: 
    path: 'creation-personnage/secrets',
    path: 'creation-personnage/races',